src/bin/index.ts

```diff
@@ -76,6 +77,11 @@ const parseCommandLineArguments = () => {
               type: "array",
               demandOption: true,
             })
+            .option("package-manager", {
+              description: "The Node package manager to use. Match this to your repository. If uncertain, prefer npm.",
```

Comment:
Opinion purely based on a [previous experience](https://github.com/guardian/cdk-playground/pull/1).

Reply:
I wonder if we should tell people how to check which package manager is in use (i.e. search for `package-lock.json` vs `yarn.lock`), just to help those who are completely unfamiliar with the Node ecosystem?

Reply:
Very minor point, but I also wonder if we should slightly rephrase `If uncertain, prefer npm`.

How about something like: `If the repository is not currently using either package manager and there is no strong convention in your team, we recommend `npm`?

Reply:
Yes, great improvement!

Reply:
Done.

Reply:
👍 great suggestions Jacob!

Perhaps another reason for suggesting `npm` is that there's probably a greater chance of it being installed in various environments if you already have Node installed? I think they're often bundled together 🤔. This may be useful for projects where there aren't existing conventions or infrastructure, e.g. Scala projects?